_data-prepper/getting-started.md

```diff
@@ -65,11 +57,12 @@ Run the following command with your pipeline configuration YAML.
 
 ```bash
 docker run --name data-prepper \
-    -v /full/path/to/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml \
+    -v /${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml \
     opensearchproject/data-prepper:latest
+    
 ```
 
```

Comment:
I have added copy buttons to Data Prepper docs that ask you to create YAML files like this, and think this section would benefit from it.

Reply:
I was waiting for someone would point that out. :)  Do I need this command? {% include copy-curl.html %}